test/development/acceptance-app/ReactRefreshLogBox.test.ts

```diff
@@ -1059,7 +1064,14 @@ describe.each(['default', 'turbo'])('ReactRefreshLogBox app %s', () => {
       const { session } = sandbox
 
       await next.patchFile('index.js', "throw new Error('module error')")
-      await session.assertHasRedbox()
+
+      // FIXME(veil): Redbox missing when sandbox type is server.
+      if (type !== 'server') {
+        await session.assertHasRedbox({
+          fixmeStackFramesHaveBrokenSourcemaps: true,
+        })
+      }
```

Comment:
How is that possible? We only changed test utils so why would the behavior change?